src/openloco/windows/CompanyFaceSelection.cpp

```diff
@@ -212,17 +192,17 @@ namespace openloco::ui::windows::CompanyFaceSelection
     // 0x004352C7
     static void scrollMouseOver(window* const self, const int16_t x, const int16_t y, const uint8_t scroll_index)
     {
-        const auto [rowIndex, object] = getObjectFromSelection(y);
+        auto [rowIndex, object] = getObjectFromSelection(y);
         if (self->row_hover == rowIndex)
         {
             return;
         }
         self->row_hover = rowIndex;
-        self->object = object._name;
-        sub_471B95();
+        self->object = &object;
```

Comment:
What I would do instead would be to pass in here the object._header ptr. You would then need to call `object_index_entry::read` to get out the other variables